runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -233,27 +235,47 @@ public void onClientTick(ClientTick clientTick)
 				}
 
 				String oldTarget = entry.getTarget();
-				String newTarget = decorateTarget(oldTarget, decorations);
+				String newTarget = decorateTarget(oldTarget, player);
```

Comment:
With this change, the guard clause for `(decorations == null)` should be moved into `decorateTarget()`.